app/budget/services/actions/budgetActions.js

```diff
@@ -39,6 +39,39 @@ class BudgetActions {
 						}
 					}
 
+					var sectionGroupCosts = results.sectionGroupCosts;
+					var terms = ["05", "07", "10", "01", "03"];
+
+					var subjectCode = results.courses[0].subjectCode;
+					var termCodes = terms.map(function(term) {
+						if (term == "01" || term == "03") {
+							return (parseInt(year) + 1) + term;
+						}
+						return year + term;
+					});
+
+					termCodes.forEach(function(termCode) {
+						DwService.getDwCensusData(subjectCode, null, termCode).then(function(census) {
+							// match courseNumber and TermCode and inject currentEnrollment number
+							const currentCensusSnapshot = census.filter(function(c) {
```

Comment:
I would avoid single character variables (with notable exceptions like 'i' where it is a common syntax for iteration counter)